c/ovm.c

```diff
@@ -705,9 +707,24 @@ static word prim_sys(int op, word a, word b, word c) {
       case 13: /* sys-closedir dirp _ _ -> ITRUE */
          closedir((DIR *)(intptr_t)cnum(a));
          return ITRUE;
-      case 14: { /* unused */
-         exit(42);
-         break; }
+      case 14: { /* get-environment-variables → alist */
```

Comment:
Parsing the environment could be done at owl side later, if this would return a pointer and there were primitives in place to read word and read byte from a pointer to owl.